test/exercism/comment_test.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../integration_helper'
+require 'minitest/pride'
+
+class CommentTest < Minitest::Test
+  include DBCleaner
+
+  attr_reader :alice, :fred, :comment, :submission
+  def setup
+    super
+    @alice = User.create(username: 'alice')
+    @fred = User.create(username: 'fred')
+    @submission = Submission.create(user: alice, language: 'python', slug: 'one')
+  end
+
+  def test_is_qualifying
+    submission.reload
+    comment = Comment.create(user: fred, submission: submission, body: 'something')
+    assert comment.qualifying?
+  end
+
+  def test_is_not_qualifying
+    submission.reload
+    comment = Comment.create(user: alice, submission: submission, body: 'something')
+    refute comment.qualifying?
+  end
```

Comment:
Why don't we have just one test?

We can do all the setup right in the test, and then have both users create a comment, one will be qualifying and one will not.

I know a lot of people say you shouldn't have more than one assertion per test, but I think that in this case it would be easier to understand the relationship if everything is in one place.